src/ImpactEvaluator.sol

```diff
@@ -37,6 +36,8 @@ contract ImpactEvaluator is AccessControl, Balances {
     receive() external payable {}
 
     function advanceRound() private {
+        uint previousRoundRemainingReward = (1 - (previousRoundTotalScores /
+            MAX_SCORE)) * previousRoundRoundReward;
```

Comment:
Isn't this expression be always rounded down to zero? I am assuming we are doing integer arithmetics here. 

```sol
previousRoundTotalScores / MAX_SCORE
```

Reply:
Great spot!

Reply:
https://github.com/Meridian-IE/impact-evaluator/pull/66